README.md

```diff
@@ -105,11 +106,21 @@ sudo dnf install ani-cli
 
 #### Arch
 
+Build and install from the AUR: 
+```sh
+yay -S ani-cli
+```
 Also consider ani-cli-git
 
+#### OpenSuse Tumbleweed
+
+First you need to add the ani-cli copr repo, then install the package:
+
 ```sh
-yay -S ani-cli
+zypper addrepo https://download.copr.fedorainfracloud.org/results/derisis13/ani-cli/opensuse-tumbleweed-x86_64/
+zypper install ani-cli
 ```
+*Note: package is noarch, so any architecture should work, even though the repo is labled x86-64*
```

Comment:
Is it possible to label the repo as noarch?
Can confirm ani-cli works on aarch64 btw